Libs/MRML/DisplayableManager/vtkThreeDViewInteractorStyle.cxx

```diff
@@ -792,6 +783,56 @@ void vtkThreeDViewInteractorStyle::SetModelDisplayableManager(
 }
 
 //----------------------------------------------------------------------------
+void vtkThreeDViewInteractorStyle::RotateToPosterior(bool inverse)
```

Comment:
You can get the Nth axis label by calling const char* vtkMRMLAbstractViewNode::GetAxisLabel(int labelIndex). If the view node is not accessible or you would always need to rotate to Nth axis then you can have a method like:
vtkThreeDViewInteractorStyle::RotateToViewAxis(int direction) // direction is in 0..5, same way as in abstract view node axis index (-X, +X, ..., -Z, +Z)
or
vtkThreeDViewInteractorStyle::RotateToViewAxis(int axis, bool positiveDirection) --- where axis is in 0..2